test/canvas-mock.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Set up mock 2D context, for usage in unit tests.
+ *
+ * Adapted from: https://github.com/Cristy94/canvas-mock
+ */
+
+var myHackedInCanvas;    // Use one global canvas instance for all calls to createElement('canvas');
```

Comment:
OK, the comment might be a bit off. It's strictly speaking not a global because it doesn't reside in the global namespace. It's a static within the module. I'll adjust the comment.

Reply:
And the name is silly; changing it to `canvasMock`.